rootfs/usr/share/inputplumber/devices/50-gpd_winmini.yaml

```diff
@@ -27,11 +27,6 @@ source_devices:
       name: "Microsoft X-Box 360 pad"
       phys_path: usb-0000:63:00.3-5/input0
       handler: event*
-  - group: keyboard
```

Comment:
Instead of removing these, can you comment  them out so the information isn't lost?